### PR TITLE
corrected/clarified the setLoopPoints() documentation

### PR DESCRIPTION
**Pull request by Coder-Rahul-Y**
The previous documentation talks about 'end points' but the current code does not take end point as an argument and hence does not allow specifying the end point of the loop. (Instead the functions allows specifying the beginning offset and the length of the loop.)

Thanks a lot for making a contribution to SFML! 🙂

Before you create the pull request, we ask you to check the follow boxes. (For small changes not everything needs to ticked, but the more the better!)

* [discussed in official Discord] Has this change been discussed on [the forum](https://en.sfml-dev.org/forums/index.php#c3) or in an issue before?
* [yes] Does the code follow the SFML [Code Style Guide](https://www.sfml-dev.org/style.php)?
* [ ] Have you provided some example/test code for your changes?
* [ ] If you have additional steps which need to be performed list them as tasks!

----

## Description

Please describe your pull request.

This PR is related to the issue #

## Tasks

* [ ] Tested on Linux
* [ ] Tested on Windows
* [ ] Tested on macOS
* [ ] Tested on iOS
* [ ] Tested on Android

## How to test this PR?

Describe how to best test these changes. Please provide a [minimal, complete and verifiable example](https://stackoverflow.com/help/mcve) if possible, you can use the follow template as a start:

```cpp
#include <SFML/Graphics.hpp>

int main()
{
    sf::RenderWindow window(sf::VideoMode(1280, 720), "Minimal, complete and verifiable example");
    window.setFramerateLimit(60);

    while (window.isOpen())
    {
        sf::Event event;
        while (window.pollEvent(event))
        {
            if (event.type == sf::Event::Closed)
                window.close();
        }

        window.clear();
        window.display();
    }
}
```

### CODE REVIEW

File: include/SFML/Audio/Music.hpp [hunk 1]
```diff
@@ -189,11 +189,11 @@ class SFML_AUDIO_API Music : public SoundStream
     TimeSpan getLoopPoints() const;
 
     ////////////////////////////////////////////////////////////
-    /// \brief Sets the beginning and end of the sound's looping sequence using sf::Time
+    /// \brief Sets the beginning and duration of the sound's looping sequence using sf::Time
     ///
-    /// Loop points allow one to specify a pair of positions such that, when the music
+    /// Loop points allow one to specify the beginning offset and the duration of the loop such that, when the music
```
Comment (eXpl0it3r) [inline include/SFML/Audio/Music.hpp:194]: Unrelated, but the `one` breaks the reading flow slightly: `Loop points allow to specify ...`

Make sure to wrap the line so it better fits the surrounding blocks.

Comment (ChrisThrasher) [reply include/SFML/Audio/Music.hpp:194]: I think "Loop points allow for specifying" would sound good

Comment (Coder-Rahul-Y) [reply include/SFML/Audio/Music.hpp]: > Unrelated, but the `one` breaks the reading flow slightly: `Loop points allow to specify ...`
> 
> Make sure to wrap the line so it better fits the surrounding blocks.

Done  ;-;

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

File: include/SFML/Audio/Music.hpp [hunk 2]
```diff
@@ -334,4 +334,4 @@ class SFML_AUDIO_API Music : public SoundStream
 ///
 /// \see sf::Sound, sf::SoundStream
 ///
-////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
```
Comment (Bromeon) [inline include/SFML/Audio/Music.hpp:337]: Why this change?

Comment (Coder-Rahul-Y) [reply include/SFML/Audio/Music.hpp]: I was fixing some stuff. It's been fixed now. 
I apologies for any inconvenience.

**Comment by Coder-Rahul-Y** [comment]
I want to target the 2.6.x branch too. Should I rebase this pull req?

**Comment by ChrisThrasher** [comment]
> I want to target the 2.6.x branch too. Should I rebase this pull req?

PRs can only target a single branch. If you want to merge this into 2.6.x and master the you'll have to make 2 PRs, but don't do that until the first is merged.

**Comment by eXpl0it3r** [comment]
Alternatively, you can rebase this PR onto `2.6.x` and change the target to `2.6.x` instead of `master` and eventually the commit will be backmerged to master

**Comment by Coder-Rahul-Y** [comment]
Successfully changed the base of this PR to `2.6.x` branch. This PR is ready for review now.

**Review by eXpl0it3r — APPROVED** [review]

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Comment by eXpl0it3r** [comment]
Thank you for making the documentation clearer 🙂
